openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/management/jvmcpumonitorbean/TestJvmCpuMonitorMXBean_Remote.java

```diff
@@ -1,179 +1,179 @@
+					leftFork = forks[i+1];
+				}
+								
+				threads[i] = new Thread(new Philosopher(name, rightFork, leftFork, size));
+			}
+			
+			for(int i=0; i < NPERSON; i++) {
+				threads[i].start();
+			}
+			
+			for(int i=0; i < NPERSON; i++) {
+				threads[i].join();
+			}
+			
+			// Calling System.GC to force garbage collection
+			System.gc();
+	
+			// Sleep for 60 secs
+			Thread.sleep(90000);
+
+			Timestamp t2 = new Timestamp(new Date().getTime());
+			
+			// Take the second snapshot of cpu information
+			cpuInfo[1] = monitorMXBean.getThreadsCpuUsage(new JvmCpuMonitorInfo());
+
+			System.out.println("TIME STAMP:" + cpuInfo[0].getTimestamp() + "," + cpuInfo[1].getTimestamp());
+			System.out.println("GC TIME:" + cpuInfo[0].getGcCpuTime() + "(" + t1 + ")" + "," + cpuInfo[1].getGcCpuTime() + "(" + t2 + ")" );
+			System.out.println("JIT TIME:" + cpuInfo[0].getJitCpuTime() + "(" + t1 + ")" + "," + cpuInfo[1].getJitCpuTime() + "(" + t2 + ")" );
+			System.out.println("APPLICATION TIME:" + cpuInfo[0].getApplicationCpuTime() + "," + cpuInfo[1].getApplicationCpuTime());
+			
+			
+			// Do the validation of data collected
+			// Check for increase of time stamp
+			assertTrue("Thread#"+ threadId + ":Snapshot timestamp is not increasing: value1 = " + cpuInfo[0].getTimestamp() + "(" + t1 + ")" + 
+					", value2 = " + cpuInfo[1].getTimestamp() + "(" + t2 + ")", (cpuInfo[0].getTimestamp() < cpuInfo[1].getTimestamp()));
+			
+			// Please note all the measured time durations are in micro seconds
+			// Check for gc cpu time
+			assertTrue("Thread#"+ threadId + ":GC cpu time is not increasing: value1 = " + cpuInfo[0].getGcCpuTime() + "(" + t1 + ")"  + 
+					", value2 = " + cpuInfo[1].getGcCpuTime() + "(" + t2 + ")", ( cpuInfo[0].getGcCpuTime() < cpuInfo[1].getGcCpuTime() ));
+							
+			//Check for jit cpu time only if jit is enabled
+			if(vmInfo.contains("JIT enabled")) {
+						
+			assertTrue("Thread#"+ threadId + ":JIT cpu time is not increasing: value1 = " + cpuInfo[0].getJitCpuTime() + "(" + t1 + ")" + 
+					", value2 = " + cpuInfo[1].getJitCpuTime() + "(" + t2 + ")", ( cpuInfo[0].getJitCpuTime() < cpuInfo[1].getJitCpuTime() ));
+			}
+			
+			// Check for System jvm cpu time
+			assertTrue("Thread#"+ threadId + ":SYSTEM JVM THREADS cpu time is not increasing: value1 = " + cpuInfo[0].getSystemJvmCpuTime() + "(" + t1 + ")" +
+					", value2 = " + cpuInfo[1].getSystemJvmCpuTime() + "(" + t2 + ")", (cpuInfo[0].getSystemJvmCpuTime() < cpuInfo[1].getSystemJvmCpuTime()));
+					
+			// Check for application cpu time
+			assertTrue("Thread#"+ threadId + ":APPLICATION cpu time is not increasing: value1 = " + cpuInfo[0].getApplicationCpuTime() + "(" + t1 + ")" +
+					", value2 = " + cpuInfo[1].getApplicationCpuTime() + "(" + t2 + ")", (cpuInfo[0].getApplicationCpuTime() < cpuInfo[1].getApplicationCpuTime()));
+
+		} catch(Exception exception) {
+			exception.printStackTrace();
+			fail("Exception thrown: " + exception.getMessage());
+		}
+	}
+}
```

Comment:
A number of the test.jlm files had Windows CRLF rather than unix line feed so I fixed them - but that caused the PR checker to insist on updating the copyright year.